frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -63,9 +64,7 @@ function FileManagerMenu:initGesListener()
 end
 
 function FileManagerMenu:setUpdateItemTable()
-    for _, widget in pairs(self.registered_widgets) do
-        widget:addToMainMenu(self.tab_item_table)
-    end
+    PluginLoader:addToMenu(self.registered_widgets, self.tab_item_table)
```

Comment:
I 100% agree with you on this. Plugins should be placed in the related menu instead of being packed into one giant menu. And when that happens, we will still need a centralized place to manage plugins (enable/disable/setting). But If we go with this design, do you think we still need this PR?

I suggest we wait for https://github.com/koreader/koreader/pull/2601 before implementing logic to support moving plugins into different menus.